tests/unit/test_network_connections.py

```diff
@@ -3260,6 +3260,15 @@ def test_set_deprecated_slave_type(self):
             self.assertTrue("port_type" in connection)
             self.assertTrue("slave_type" not in connection)
 
+    def test_mac_address_argvalidator(self):
```

Comment:
This should probably be updated to test for ethernet or infiniband or both.

Reply:
We can not know the connection type inside `ArgValidatorMac`, it may not be  achievable to have the unit tests for ethernet or infiniband respectively(based on connection type),

But we still have the sanity check which based on connection type (ethernet or infiniband)  
https://github.com/linux-system-roles/network/blob/main/module_utils/network_lsr/argument_validator.py#L1484